tensorflow/lite/model.cc

```diff
@@ -627,6 +627,12 @@ TfLiteStatus InterpreterBuilder::operator()(
     return kTfLiteError;
   }
 
+  if (num_threads < -1) {
+    error_reporter_->Report(
+        "num_threads should be greater than -1(default value).");
```

Comment:
Is the error msg somewhat confusing here as "-1" is actually a signal to indicate to use a (default value for #threads) which is set by the particular context?

Reply:
@multiverse-tf : Yes, I agree, the error message is confusing.
How about "num_threads should be greater than -1 or equal to -1 to reset to default value." 
Please suggest to better. TIA!

Reply:
"num_threads should be >=0 or just -1 to let TFLite runtime set the value."?

Reply:
Thanks! This looks better!